packages/snaps-execution-environments/src/index.ejs

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+
+  <title>MetaMask Snaps execution environment</title>
```

Comment:
Not sure if any testing relies on this title to be the same as before 😄

Reply:
Updated to be the same.